src/Block/AuditBlockService.php

```diff
@@ -39,16 +39,21 @@ public function __construct(Environment $twig, AuditReader $auditReader)
 
     public function execute(BlockContextInterface $blockContext, ?Response $response = null): Response
     {
+        $template = $blockContext->getTemplate();
+        \assert(null !== $template);
+        $limit = $blockContext->getSetting('limit');
```

Comment:
maybe cast is better here?

Reply:
Since there is
```
public function configureSettings(OptionsResolver $resolver): void
    {
        $resolver->setDefaults([
            'limit' => 10,
            'template' => '@SonataDoctrineORMAdmin/Block/block_audit.html.twig',
        ]);
    }
``` 

What do you think about using `allowedTypes` ?

Reply:
Is that a feature from the options resolver? I think it would be great but not sure if phpstan will understand it tho.

Reply:
Phpstan won't understand, so I'll keep the `assert`, but it still a code improvement